util/tooltool/tooltool.py

```diff
@@ -771,9 +774,9 @@ def _send_batch(base_url, auth_file, batch, region):
     return json.load(resp)['result']
 
 
-def _s3_upload(filename, file):
+def _s3_upload(filename, actualFile):
```

Comment:
The convention I've seen (and used) is to make `file` ➡️ `file_` to avoid the built-in name.